modules/nvidia_plugin/tests/functional/shared_tests_instances/set_device_name.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "set_device_name.hpp"
+#include <string>
+#include <stdexcept>
+
+namespace ov {
+namespace test {
+
+void set_device_suffix(const std::string& suffix) {
+    if (!suffix.empty()) {
+        throw std::runtime_error("The suffix can't be used for NVIDIA GPU device!");
```

Comment:
It does not look like a test ... just some setter with exception.
How test for `set_device_suffix` should look like ?

Reply:
Do you really think that I am going to break CI? :D
I am interested in green CI and fully parity all plugins with OpenVINO.

it's not test. I referred to OpenVINO PR in the description. This code connected with IE test infra only and not connected with the plugin test instantiations. 
Short description: this functionality allows to reuse plugin functional tests for other devices covered by the plugin (for example: GPU.0 = iGPU, GPU.1 (intel dGPU) and so on)
@vladimir-paramuzov requested such changes